test/modules/PlugAndPlayDevice/PlugAndPlayIdentity.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you also exclude your module from code coverage by adding it to `CodeCoverage.runsettings`? I think the following will work, somewhere in [this section](https://github.com/Azure/iotedge/blob/1017802/CodeCoverage.runsettings#L57):

```
<ModulePath>.*plugandplaydevice\..*</ModulePath>
```